app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -202,6 +202,14 @@ def build_metric_id(type, resource, metric_id)
       "#{type}I~R~[#{resource[:middleware_server][:feed]}/#{resource[:nativeid]}]~#{type}T~#{metric_id}"
     end
 
+    def self.update_alert(*args)
```

Comment:
Yes, the following PR linked with this work will implement those methods.
It looks like a skeleton now but motivation was to split this feature for easy review.